File: app/src/main/java/com/wang17/myclock/model/Commodity.java
package com.wang17.myclock.model;

public class Commodity {
    public Commodity(String item, String name, int unit, double cose) {
        this.item = item;
        this.name = name;
        this.unit = unit;
        this.cose = cose;
    }

    // 合约
    public String item;
    // 名称
    public String name;
    // 交易单位  比如：1000克/手
    public int unit;
    // 最小变动价位 0.05元/克
    public double cose;
}

File: app/src/main/java/com/wang17/myclock/model/Commodity.kt
package com.wang17.myclock.model

class Commodity {
    // 合约
    var item: String=""
    // 名称
    var name: String=""
    // 交易单位  比如：1000克/手
    var unit: Int=0
    // 最小变动价位 0.05元/克
    var cose: Double=0.0

    constructor(item:String,name:String,unit:Int,cose:Any){
        this.item = item
        this.name=name
        this.unit=unit
        this.cose = cose.toString().toDouble()
    }
}